src/main/java/com/vmware/vipclient/i18n/messages/service/ComponentsService.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2019 VMware, Inc.
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package com.vmware.vipclient.i18n.messages.service;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.vmware.vipclient.i18n.messages.api.opt.server.ComponentsBasedOpt;
+import com.vmware.vipclient.i18n.messages.dto.MessagesDTO;
+import com.vmware.vipclient.i18n.util.ConstantsKeys;
+
+public class ComponentsService {
+
+	Logger logger = LoggerFactory.getLogger(ComponentsService.class);
+	private final List<String> components;
+	private final List<String> locales;
+
+	/**
+	 * @param components
+	 * @param locales
+	 */
+	public ComponentsService(List<String> components, List<String> locales) {
+		this.components = components;
+		this.locales = locales;
+	}
+
+	public Map<String, Map<String, Map<String, String>>> getTranslation() {
```

Comment:
add description to this method to make it more easily understood.

Reply:
Fixed.